### ISSUE TITLE
windows installer - firewall entry

### ISSUE THREAD

**Issue by thezoggy** [sabnzbd/sabnzbd#2528]
While looking at something I noticed that there is a lot of firewall entries for sab/sab-console:
![mmc_2023-04-11_17-06-15](https://user-images.githubusercontent.com/500882/231298181-fc905c58-a980-4c6a-9727-0d1d6d03046d.png)

I assume this is just fallout from each time I've used the windows installer and it just add a new Inbound Rule entry regardless if it already exists.. I have like 120 of them for each.  Cleaning up rules manually, just tossing this here to remind myself to look after next install/dig more into this

**Comment by thezoggy** [comment]
quick glance, i see firewall rules are being added:
https://github.com/sabnzbd/sabnzbd/blob/develop/builder/win/NSIS_Installer.nsi#L183

liteFirewallW is pretty old and was the best one to use back in the day.
found the source at: https://bitbucket.org/lsun/litefirewall/src/master/
and looks like the 'latest' release from 2011 is the same version we are using(hashes match)

I found mozilla actually added a modified for this plugin here to solve the issue of prevent adding rule if existing is found:
https://github.com/mozilla/newtab-dev/blob/master/other-licenses/nsis/Contrib/liteFirewall/liteFirewall.cpp#L127-L139

So I guess we could prob copy that hack (and not do their other mods) and have an updated version that fixes it..

While poking around I see that "NSIS Simple Firewall" looks to be the one to use now days as unicode support was finally added in 2021 and somewhat still maintained: https://nsis.sourceforge.io/NSIS_Simple_Firewall_Plugin

"SimpleFC.dll" is much larger and has a lot more features.. but is it worth it to bloat our installer? probably not, since I would think it would take a long time + lot of installs to have so many firewall rules to do any harm. 

So anyways I'll leave this here for now in case anyone gets bored and compile a new liteFirewallW.dll
